extension/js/agent/utils/toJson.js

```diff
@@ -1,10 +0,0 @@
-var toJSON = function(obj) {
-  try {
-    return JSON.parse(JSON.stringify(obj));
-  }
-  catch (e) {
-    debug.log('failed parsing object', obj);
-    return {};
-  }
-};
-
```

Comment:
Yeah I was on the fence about moving this one, but it seemed similar to some of the other formatters and `typeof` that was only available to `inspectValue`